jawa/cf.py

```diff
@@ -189,7 +189,8 @@ def version(self):
         return self._version
 
     @version.setter
-    def version(self, (major, minor)):
+    def version(self, major_minor):
+        major, minor = major_minor
         self._version = ClassVersion(major, minor)
```

Comment:
No point making the two assignments here, just `ClassVersion(major_minor[0], major_minor[1])`

Reply:
Was going based on [PEP 3113](https://www.python.org/dev/peps/pep-3113/), which recommended that for transition.  This does have the benefit of directly erroring when more than 2 values are passed in, but that's not too importing; will change it.